Source/SpeechToTextV1/Models/SpeechRecognitionResults.swift

```diff
@@ -46,5 +48,11 @@ public struct SpeechRecognitionResults {
             results.append(wrapper.results[wrapperIndex])
             wrapperIndex += 1
         }
+        /// If we have parsed some speakerLabel objects, then store them here
```

Comment:
This is a bit nitpicky, but let's start this comment with 2 slashes `//` instead of 3 `///`, since 3 slashes is only necessary for public comments.